blackjack.py

```diff
@@ -99,23 +96,25 @@ def check_natural_win(self):
             return len(vals) == 2 and 'Ace' in vals and vals < {'Ace', 'Jack', 'King', 'Queen'}
         return False
 
-    def get_soft_hand_value(self):
+    @property
+    def soft_hand_value(self):
         """
         Calculates hand value with Ace as 1
         :return: int
         """
-        return (21.1 if self.check_natural_win()
+        return (21.1 if self.is_natural_win()
                 else sum(card.points for card in self.hand))
 
-    def get_hand_value(self):
+    @property
+    def hand_value(self):
         """
         Calculates hand value with any Ace as 11 as long as it doesn't lead to bust
         :return: int
         """
-        if self.check_natural_win():
+        if self.is_natural_win():
             return 21.1
         else:
-            hand_total = self.get_soft_hand_value()
+            hand_total = self.soft_hand_value
```

Comment:
What I don't like here is that you'll be calling `self.is_natural_win` a second time for nothing.
Did you see my suggestion in an earlier comment?